components/ILIAS/Forum/PRIVACY.md

```diff
@@ -0,0 +1,101 @@
+missing or wrong information using the [ILIAS issue tracker](https://mantis.ilias.de)
+or contribute a fix via [Pull Request](../../../docs/development/contributing.md#pull-request-to-the-repositories).
+
+## General Information
+
+The forum is an ILIAS repository object in which users can hold discussions. A forum is organized in different topics (threads), to which contributions (posts) are written. As long as a forum is not deleted, posts can be viewed for years after they have been published.
+
+At least one person is responsible for moderation in all forums: They can censor and overwrite posts and close topics. Moderators can be added and removed by a person with the “Edit Settings” permission on the “Moderators” tab.
+
+## Configuration 
+
+Various privacy-related settings are made under “Administration > Repository and Objects > Forum”:
+
+**Enable Statistics in Forum** (default: off) sets for the installation whether a statistics function can be activated in the settings of individual forums.
+
+**Allow posting with pseudonym** (default setting: off) sets whether pseudonymization can be activated in individual forums. Registered users can then post in this forum using an alias without stating their name. 
+
+**Attachment** (default setting: on) sets whether file attachments are possible for posts in all forums or whether this function must be activated in individual forums.
+
+**Notifications** (default setting: on, without file attachments) sets whether users can be informed by e-mail about new posts in forums and whether these e-mails can also contain file attachments. The notification can be sent immediately or once a day if the cron job “Send Forum Notifications via Cron Job” is activated.
+
+**Drafts** (default setting: off) sets whether it is possible to save draft posts in the forum and at what intervals drafts are automatically saved.
+
+The following privacy-related settings can be made in an individual forum under “Settings”:
+
+### Basic Settings
+
+**User Functions / Thread Rating** (default: off) enables users to assign a 5-star rating to threads. The average rating and number of ratings is displayed to all users.
+
+**Moderator Functions / Approve Postings** (default: off) enables posts in the forum to be published only after they have been read and approved by moderators.
+
+**Privacy / Enable Statistics** (default: off) enables all forum users to see who has published how many posts.
+
+**Privacy / Posting with pseudonym** (default setting: off) allows users to publish posts in the forum anonymously or using a pseudonym. Forum posts that have already been anonymized remain anonymous even after a changeover.
+
+### Notifications
+
+For forums within courses and groups, you can set here whether users of the forum automatically receive notifications by e-mail, whether they have to activate it manually or whether they can deactivate it individually (default setting: manual activation).
```

Comment:
For clarification you can change to:
"For forums within courses and groups, you can set whether users automatically receive notifications by e-mail, need to activate them manually, or can deactivate them individually (default setting: manual activation)."